src/pages/workspaces/workspace/analysis/modals/CloudEnvironmentModal.js

```diff
@@ -207,9 +207,9 @@ export const CloudEnvironmentModal = ({
           shape: 'pause',
           toolLabel,
           onClick: () => stopApp(toolLabel),
-          disabled: busy || !canCompute,
+          disabled: cloudProvider === cloudProviderTypes.AZURE || busy || !canCompute, //TODO: IA-3993 enable pausing
```

Comment:
Nit: I don't know if this is preferred, but I think it might be cleaner to use the `isAzureWorkspace(workspace)` method from workspace-utils. We seem to use that method a lot and it might be nice for consistency. Maybe @jdcanas has a take?

Reply:
yeah I'd much prefer that!

Reply:
this and the others were implemented as of the most recent push